src/pixi.inspector.js

```diff
@@ -19,7 +19,7 @@
 	}
 
 	var inspector = {
-		
```

Comment:
no worries, its easier if we use ?w=1 :D
(https://github.com/bfanger/pixi-inspector/pull/10/files?w=1)